triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -15,11 +15,14 @@ case class ServiceConfig(
     httpPort: Int,
     ledgerHost: String,
     ledgerPort: Int,
+    maxInboundMessageSize: Int,
     timeProviderType: TimeProviderType,
     commandTtl: Duration,
 )
 
 object ServiceConfig {
+  val DefaultMaxInboundMessageSize: Int = 4194304
```

Comment:
You probably want to get this from `RunnerConfig.DefaultMaxInboundMessageSize` once you add it to the standalone runner instead of having the default specified twice.

Reply:
✔️